tools/webpack

```diff
@@ -55,7 +57,27 @@ def build_for_dev_server(host, port, minify, disable_host_check):
         webpack_args.append('--optimize-minimize')
     if disable_host_check:
         webpack_args.append('--disable-host-check')
-    os.execvp(webpack_args[0], webpack_args)
+    webpack_process = subprocess.Popen(webpack_args)
```

Comment:
I’d move this into the `try` block to close a silly race at startup. (Yes that also opens a different silly race, but the different silly race has a more obvious failure mode.)

Reply:
Can you explain why there would be race conditions?

Reply:
In the old version, if we started `webpack` outside the `try` block, and we got a `KeyboardInterrupt` between starting `webpack` and entering the `try` block, we wouldn’t stop `webpack`. (This probably wasn’t likely to happen in practice, but if we can protect against it by reordering some statements, why not.)